src/index.js

```diff
@@ -56,9 +57,10 @@ export default function supernova(env) {
             selectionsAPI,
             muiParameters,
             theme,
+            cellLockedImage,
           });
         }
-      }, [tableData, constraints, selectionsAPI.isModal()]);
+      }, [tableData, constraints, selectionsAPI.isActive()]);
```

Comment:
This should be isModal. Currently it's not working in sense, but I have a fix for that in sense-client on the way in

Reply:
okay, when that fix is done, please let me know. I will change it to isModal.

Reply:
You can change back now